docs/src/token.mdx

```diff
@@ -1819,11 +1819,12 @@ The sender's wallet must not require that the recipient's main wallet address
 hold a balance before allowing the transfer.
 
 ### Registry for token details
+
 At the moment there exist two solutions for Token Mint registries:
 
-* hard coded addresses in the wallet or dapp
-* [spl-token-registry](https://www.npmjs.com/package/@solana/spl-token-registry)
-package, maintained at [https://github.com/solana-labs/token-list](https://github.com/solana-labs/token-list)
+* Hard coded addresses in the wallet or dapp
+* Metaplex Token Metadata. Learn more at the
+[Token Metadata Documentation](https://docs.metaplex.com/programs/token-metadata/)
```

Comment:
Do we want to link to https://github.com/solana-labs/token-list still from here? It has links and instructions to use some tools to create a token with metadata that might be more helpful than just Metaplex docs for people who aren't familiar with the history of the token list.

Reply:
That's a good point... ok, i'll put it back with an EOL warning. Even months later, many people have linked the token-list repo in questions without reading the warning at the top